nether/portal.lua

```diff
@@ -29,6 +29,27 @@ do
 	end
 end
 
+-- Nether aware mods will need to know if a player is in the nether.
+nether['is_player_in_nether'] = function (player)
+	local pname = player:get_player_name()
+	if players_in_nether[pname] == nil then
+		return false
+	else
+		return true
+	end
```

Comment:
The longer form is more readable at a glance. I generally try to avoid cramming too much into one line, but this is your mod, so of course I will adhere to your style guidelines.